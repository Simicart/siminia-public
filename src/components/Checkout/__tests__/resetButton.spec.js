import React from 'react';
import testRenderer from 'react-test-renderer';
import ResetButton from '../resetButton';

jest.mock('src/classify');

test('renders a ResetButton component', () => {
    const props = {
        resetCheckout: jest.fn()
    };
    const component = testRenderer.create(<ResetButton {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
});
