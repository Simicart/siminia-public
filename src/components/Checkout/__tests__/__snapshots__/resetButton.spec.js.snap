// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a ResetButton component 1`] = `
<button
  className="root_normalPriority"
  onClick={[MockFunction]}
  type="button"
>
  <span
    className="content"
  >
    Continue Shopping
  </span>
</button>
`;
